docs/sources/operations/autoscaling_queriers.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Autoscaling Loki queriers
+menuTitle: Autoscaling queriers
+description: Kubernetes deployments of a microservices mode Loki cluster can use KEDA to autoscale the quantity of queriers.
+weight: 30
+---
+
+# Autoscaling Loki queriers
+
+A microservices deployment of a Loki cluster that runs on Kubernetes typically handles a
+workload that varies throughout the day.
+To make Loki easier to operate and optimize the cost of running Loki at scale,
+we have designed a set of resources to help you autoscale your Loki queriers.
+
+## Prerequisites
+
+You need to run Loki in Kubernetes as a set of microservices. You need to use the query-scheduler.
+
+We recommend using [Kubernetes Event-Driven Autoscaling (KEDA)](https://keda.sh/) to configure autoscaling
+based on Prometheus metrics. Refer to [Deploying KEDA](https://keda.sh/docs/latest/deploy) to learn more
+about setting up KEDA in your Kubernetes cluster.
+
+## Scaling metric
+
+Because queriers pull queries from the query-scheduler queue and process them on the querier workers, you should scale metrics based on:
+
+- The scheduler queue size.
+- The queries running in the queriers.
+
+The query-scheduler exposes the `cortex_query_scheduler_inflight_requests` metric.
+It tracks the sum of queued queries plus the number of queries currently running in the querier workers.
+The following query is useful to scale queriers based on the inflight requests.
+
+```promql
+sum(
+  max_over_time(
+    cortex_query_scheduler_inflight_requests{namespace="loki-cluster", quantile="<Q>"}[<R>]
+  )
+)
+```
+
+Use the quantile (Q) and the range (R) parameters to fine-tune the metric.
+The higher Q is, the more sensitive the metric is to short-lasting spikes.
+As R increases, we reduce the variation over time in the metric.
+A higher R-value helps avoid the autoscaler from modifying the number of replicas too frequently.
+
+In our experience, we have found that a Q of 0.75 and an R of 2 minutes work well.
+You can adjust these values according to your workload.
+
+## Cluster capacity planning
+
+When configuring KEDA to scale your Loki queriers, we need to configure:
+
+- The threshold for scaling up and down
+- The scale down stabilization period
+- The minimum and the maximum number of queriers
+
+Querier workers process queries from the queue. You can configure each Loki querier to run several workers.
+To reserve workforce headroom to address workload spikes, our recommendation is not to use more than 75% of the workers.
+For example, if you configure the Loki queriers to run 6 workers, set a threshold of `floor(0.75 * 6) = 4`.
+
+Our recommendation for the minimum number of queriers is to run at least one querier, and to look at the average
+number of inflight requests 75% of the time in the last seven days, targeting a 75% utilization of the queriers.
```

Comment:
```suggestion
To determine the minimum number of queries that you should run, run at least one querier for seven days and determine the average
number of inflight requests the system processes 75% of the time. The target utilization of the queries is 75%.
```

Reply:
> run at least one querier for seven days

This doesn't sound correct to me. Our recommendation is to run at least one querier (hence the `clamp_min`). but running one querier for seven days.

Reply:
@salvacorts , updated the suggestion. Does that wording work better?

Reply:
Works perfectly! Thank you :)